_docs/reference/release-notes.md

```diff
@@ -1,13 +1,84 @@
 ---
 title: Release Notes
-overview: What's been happening with Istio.
+overview: What's been happening with Istio
 
 order: 50
 
 layout: docs
 type: markdown
 ---
 
+## Istio 0.2
+
+#### General
+
+- **Updated Config Model**. Istio now uses the Kubernetes Custom Resource Definition model to describe and store its configuration.
+When running in Kubernetes environment, configuration can now be optionally managed using the `kubectl` command.
+
+- **Multiple Namespace Support**. Istio control plane components are now in the dedicated "istio-system" namespace. Istio can manage 
+services in other non-system namespaces.
+
+- **Mesh Expansion**. Initial support for adding non-Kubernetes services (in the form of VMs and/or physical machines) to a mesh. This is an early version of
+this features which has limitations (such as requiring a flat network across containers and VMs).
+
+- **Multi-Platform Support**. Initial support for running Istio outside Kubernetes using Consul or Eureka as service registries. This feature can handle 
```

Comment:
Should we call this Multi-Environment support?  We're using Environment in all our other docs to describe this...maybe keep consistent?

Reply:
I might avoid the phrasing 'running Istio outside', perhaps 'using Istio in conjunction with other service registries...'